packages/react-admin/package.json

```diff
@@ -9,6 +9,7 @@
         "src"
     ],
     "main": "lib/index",
+    "jsnext:main": "src/index",
```

Comment:
Nice addition, however not related to this PR

Reply:
Will revert and PR in other request